scripts/prepare_sector_network.py

```diff
@@ -2286,6 +2334,73 @@ def add_biomass(n, costs):
         e_initial=solid_biomass_potentials_spatial,
     )
 
+    if snakemake.params.biomass.get(
+        "include_unsustainable"
+    ) and investment_year < snakemake.params.biomass.get("unsustainable_phase_out"):
+
+        n.madd(
+            "Store",
+            spatial.gas.biogas,
+            suffix=" unsustainable",
+            bus=spatial.gas.biogas,
+            carrier="unsustainable biogas",
+            e_nom=unsustainable_biogas_potentials_spatial,
+            marginal_cost=costs.at["biogas", "fuel"],
+            e_initial=unsustainable_biogas_potentials_spatial,
+            efficiency=1,
+            e_min_pu=0,
+            e_nom_extendable=False,
+        )
+
+        n.madd(
+            "Store",
+            spatial.biomass.nodes,
+            suffix=" unsustainable",
+            bus=spatial.biomass.nodes,
+            carrier="unsustainable solid biomass",
+            e_nom=unsustainable_solid_biomass_potentials_spatial,
+            marginal_cost=14,  # mean of MINBIOWOO costs in ENS_BAU 2030
+            e_initial=unsustainable_solid_biomass_potentials_spatial,
+            efficiency=1,
+            e_min_pu=0,
+            e_nom_extendable=False,
+        )
+
+        n.madd(
+            "Bus",
+            spatial.bioliquids.nodes,
+            location=spatial.bioliquids.locations,
+            carrier="unsustainable bioliquids",
+            unit="MWh_LHV",
+        )
+
+        n.madd(
+            "Store",
+            spatial.bioliquids.nodes,
+            suffix=" unsustainable",
+            bus=spatial.bioliquids.nodes,
+            carrier="unsustainable bioliquids",
+            e_nom=unsustainable_liquid_biofuel_potentials_spatial,
+            marginal_cost=costs.at["oil", "fuel"],
```

Comment:
does it make sense to assume the same cost as fossil oil? what does the JRC ENSPRESO database say on that? I guess if the costs are higher than fossil oil, you wouldn't see the level of bioliquids used today when the global CO2 constraint is not binding. Is that the reason?

Reply:
Fossil oil 2020 has fuel costs of 52.91 €/MWh, MINBIOCRP11 (Bioethanol barley, wheat, grain maize, oats, other cereals and rye) in ENSPRESO a mean of 81.36 €/MWh (2030 for BAU scenario). Setting p_max_pu=0 for the respective Stores should force its usage despite the higher costs. I could add another row in the technology-data. However, the ENSPRESO cost value does not include the costs incurred for the intemediary processing step from crops to ethanol.

Reply:
You mean `e_max_pu`, right?

It's already better than assuming fuel cost of fossil oil, so I would be fine like this. Is there a cost (like VOM) for BtL you could perhaps used?

Reply:
Yes sorry, I meant `e_max_pu`!

Good idea, I could assign the VOM value of BtL to the link to the oil bus or add it on top of the marginal cost of the Store.

Reply:
on the link, I would say